.github/workflows/beaker.yml

```diff
@@ -72,7 +77,7 @@ jobs:
         if: ${{ inputs.rubocop }}
       - name: Setup Test Matrix
         id: get-outputs
-        run: bundle exec metadata2gha --use-fqdn --pidfile-workaround ${{ inputs.pidfile_workaround }}
+        run: bundle exec metadata2gha --domain ${{ inputs.domain }} --pidfile-workaround ${{ inputs.pidfile_workaround }}
```

Comment:
This requires at least puppet_metadata 1.10.0. Can you change the example in https://github.com/voxpupuli/gha-puppet#gemfile-integration-example to be `~> 1.10` instead of `~> 1.0`?

Reply:
updated it